src/services/ui/src/components/Opensearch/main/Filtering/FilterChips.tsx

```diff
@@ -14,17 +14,10 @@ interface RenderProp {
   clearFilter: (filter: opensearch.main.Filterable, valIndex?: number) => void;
 }
 
-// simple date range chips
-const DateChip: FC<RenderProp> = ({
-  filter,
-  index,
-  openDrawer,
-  clearFilter,
-}) => {
+const ChipDate: FC<RenderProp> = ({ filter, openDrawer, clearFilter }) => {
   const value = filter.value as opensearch.RangeValue;
   return (
     <Chip
-      key={`${index}-${filter.field}`}
```

Comment:
Noticed the key error was being triggered in the console.  Fixed it by moving the key assignment down to the mapping.